src/modules/commander/Arming/ArmAuthorization/ArmAuthorization.cpp

```diff
@@ -79,12 +71,93 @@ static uint8_t (*arm_check_method[ARM_AUTH_METHOD_LAST])() = {
 	_auth_method_two_arm_check,
 };
 
+union Deprecated_arm_parameters {
+	struct {
+		uint8_t authorizer_system_id;
+		uint16_t auth_method_arm_timeout_msec;
+		uint8_t authentication_method;
+	} __attribute__((packed)) struct_value;
+	int32_t param_value;
+};
+
+bool _first_param_update{true};
+
+void arm_auth_param_update()
+{
+	//get new and deprecated parameters
```

Comment:
Probably wasn't clear, but you can do the transition directly on param import: https://github.com/PX4/Firmware/blob/master/src/lib/parameters/param_translation.cpp#L42. Then you can remove the previous parameter definition completely.

Reply:
I do see how to do it in `param_transtation` as I not only need to rename a parameter and convert its value but also to change 2 other parameters.
How can I acces other parameters in `param_transtation` ?

Reply:
Using `param_find_no_notification` and `param_set_no_notification` should just work in there as well.

Reply:
of course!
That's done!